### PR TITLE
Corrected pre-run checks for the volume rebalance flows

### PR DESCRIPTION
**Pull request by shtripat**
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by r0h4n** [comment]
Need an issue for this

File: tendrl/gluster_integration/objects/volume/atoms/rebalance_not_running/__init__.py [hunk 1]
```diff
@@ -26,35 +26,34 @@ def run(self):
...
+                    node_id=key.split('/')[-1]
+                ).load()
+                if obj.rebal_status is not None:
+                    if obj.rebal_status == "in progress":
+                        Event(
+                            Message(
+                                priority="info",
+                                publisher=NS.publisher_id,
+                                payload={
+                                    "message": "Volume rebalance status is running on node %s" %
```
Comment (r0h4n) [inline tendrl/gluster_integration/objects/volume/atoms/rebalance_not_running/__init__.py]: "Volume rebalance is in progress on node %s"

File: tendrl/gluster_integration/objects/volume/atoms/rebalance_running/__init__.py [hunk 2]
```diff
@@ -26,32 +26,34 @@ def run(self):
...
+                ).load()
+                if obj.rebal_status is not None:
+                    if obj.rebal_status == "in progress":
+                        return True
+            Event(
+                Message(
+                    priority="info",
+                    publisher=NS.publisher_id,
+                    payload={
+                        "message": "No rebalance running for volume %s" %
```
Comment (r0h4n) [inline tendrl/gluster_integration/objects/volume/atoms/rebalance_running/__init__.py]: "No rebalance in progress for volume %s"

File: tendrl/gluster_integration/objects/volume/atoms/rebalance_not_running/__init__.py (hunk 1 above)
Comment (shtripat) [inline tendrl/gluster_integration/objects/volume/atoms/rebalance_not_running/__init__.py]: Done

File: tendrl/gluster_integration/objects/volume/atoms/rebalance_running/__init__.py (hunk 2 above)
Comment (shtripat) [inline tendrl/gluster_integration/objects/volume/atoms/rebalance_running/__init__.py]: Done

**Comment by shtripat** [comment]
Not verified yet. Also this PR might change if we maintain the rebalance status at volume level. Will test and update the PR accordingly

File: tendrl/gluster_integration/objects/volume/atoms/rebalance_not_running/__init__.py (hunk 1 above)
Comment (r0h4n) [inline tendrl/gluster_integration/objects/volume/atoms/rebalance_not_running/__init__.py]: Is this done?

File: tendrl/gluster_integration/objects/volume/atoms/rebalance_not_running/__init__.py [hunk 3]
```diff
@@ -26,35 +26,34 @@ def run(self):
             )
         )
         try:
-            rebal_status = NS._int.client.read(
-                'clusters/%s/Volumes/%s/rebal_status' % (
+            vol_rebal_det = NS._int.client.read(
+                'clusters/%s/Volumes/%s/RebalanceDetails' % (
```
Comment (r0h4n) [inline tendrl/gluster_integration/objects/volume/atoms/rebalance_not_running/__init__.py]: Why raw read?

File: tendrl/gluster_integration/objects/volume/atoms/rebalance_running/__init__.py (hunk 2 above)
Comment (r0h4n) [inline tendrl/gluster_integration/objects/volume/atoms/rebalance_running/__init__.py]: Is this done?

**Comment by r0h4n** [comment]
Resolve conflicts @shtripat

**Comment by shtripat** [comment]
I feel this PR is not required anymore with recent changes. @nnDarshan can you please confirm?

**Comment by nnDarshan** [comment]
@shtripat Ack. Its no longer needed as we get volume leval rebalance status

**Comment by shtripat** [comment]
Closing as confirmed by @nnDarshan

**Pull request closed without merging**